pkg/client/unversioned/jobs.go

```diff
@@ -85,7 +89,7 @@ func (c *jobs) Delete(name string, options *api.DeleteOptions) (err error) {
 		return c.r.Delete().Namespace(c.ns).Resource("jobs").Name(name).Do().Error()
 	}
 
-	body, err := api.Scheme.EncodeToVersion(options, c.r.APIVersion())
+	body, err := api.Scheme.EncodeToVersion(options, latest.GroupOrDie("").GroupVersion)
```

Comment:
Why is this change necessary? Scanning other client implementations I see `c.client.APIVersion()` or `c.r.APIVersion()` but never `latest.GroupOrDie`